cookie_access_test.html

```diff
@@ -32,6 +36,34 @@
         return "";
     }
 
+    // Allowed test origins
+    // * senglehardt.com is here temporarily until we move the
+    //   test suite on that domain to a Mozilla-owned domain
+    // * several Pocket origins for the ETP user study
+    var testOrigins = [
+      'senglehardt.com',
+      'pocket.com',
+      'getpocket.com',
+      'featfront-67etp-study.web.readitlater.com',
+      'etpstudy.web.readitlater.com'
+    ];
+
+    // Try to set a cookie
+    var randVal = Math.random();
+    var randVal2 = 'test_' + Math.random();
+    document.cookie = randVal2+'='+randVal;
+
+    // Return result
+    if ('test_origin' in qs && testOrigins.includes(qs['test_origin'])) {
```

Comment:
I assume the including pages are also being updated to include `test_origin` querystring param in their `iframe src`?

Reply:
Yes they will. I haven't yet updated my test pages but I will when list lands.